src/quo/components/wallet/network_routing/view.cljs

```diff
@@ -86,10 +86,10 @@
 
 (defn- dashed-line
   [network-name]
-  [rn/view {:style style/dashed-line}
-   (take 19
-         (interleave (repeat [rn/view {:style (style/dashed-line-line network-name)}])
-                     (repeat [rn/view {:style style/dashed-line-space}])))])
+  (into [rn/view {:style style/dashed-line}]
```

Comment:
FYI: I resolved almost all or all (can't remember now) of the React warnings about non-unique keys. This is one example.